docs/source/troubleshooting/mistral.rst

```diff
@@ -21,6 +21,7 @@ When that happens, the new database tables, relationships, and indices must be d
 ``mistral-db-manage upgrade head`` command can be re-run. For more details, review the specific
 section for the version being upgraded in :doc:`/install/upgrades`.
 
+.. _mistral-workflows-latency:
```

Comment:
Linking from the mistral page to mistral troubleshooting page.

@LindsayHill Please take a look once again if this construct something that'll work. Took an example from another page.

Reply:
Yes, that's the right sort of construct to use for cross-reference